particle/development/approach.md

```diff
@@ -1,16 +1,51 @@
 # Approach
 
+## Assets
```

Comment:
Is it worth explaining `path.assets` here?

Reply:
This is another one that I would request someone else add in as I'm not familiar with that part of Particle yet